examples/riot/buildAll.sh

```diff
@@ -21,8 +21,3 @@ for board in "${BOARDS[@]}"; do
 			popd
 	done
 done
-
-# Build lf example
-pushd hello_lf
-run/build.sh
-popd
```

Comment:
Integrating the LFC into the make file is really nice! Now we don't need to handle _LF examples different from c-code based examples anymore!